packages/marko-web-identity-x/components/form-login.marko

```diff
@@ -0,0 +1,3 @@
+<marko-web-identity-x-context|{ user }|>
+  <marko-web-browser-component name="IdentityXForm" props={ activeUser: user, context: 'login' } />
```

Comment:
How can a website override these forms?

Reply:
Added ability to customize and instructions to Readme

Reply:
Added ability to customize and instructions to Readme